cumulus/polkadot-parachain/src/chain_spec/yet_another_parachain.rs

```diff
@@ -0,0 +1,114 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! ChainSpecs dedicated to parachain setups for testing and example purposes
+
+use crate::chain_spec::get_from_seed;
+use parachains_common::genesis_config_helpers::get_account_id_from_seed;
+use polkadot_omni_node_lib::chain_spec::{Extensions, GenericChainSpec};
+use sc_chain_spec::ChainType;
+use sp_core::{
+	sr25519::{self, Pair as SrPair},
+	Pair,
+};
+use yet_another_parachain_runtime::AuraId;
+
+const NUM_ACCOUNT_PAIRS: usize = 16000;
```

Comment:
Would 12k suffice? (Given our current PoV size limitations, might be beneficial for the community spamming chains.